twingly_search/query.py

```diff
@@ -1,120 +0,0 @@
-import datetime
-from pytz import utc
-
-from twingly_search.errors import TwinglyQueryException
-
-try:
-    # python 3
-    from urllib.parse import urlencode
-except ImportError:
-    from urllib import urlencode
-
-
-class Query(object):
```

Comment:
We would like to still have the query class, just as we have in the Ruby implementation, even though the URL query parameters has been removed in the new version of the API.
Keeping things compatible with the old version to 100% isn't necessary, but we would still like to have the Query class for the following reasons:

* Without having the `Query#start_time` and `Query#end_time` helper methods, the user has to think about formatting (and possibly quoting) the date correctly. This makes it easier for us to give good error messages to the user when an invalid date is set in the query.
* The `start_time` and `end_time` helpers also makes it easier to paginate through multiple pages of results using `start-time` without having to rebuild the query string themselves with a new date each time. (pagination is explained [here](https://developer.twingly.com/resources/search/#pagination). The `find_all_posts_mentioning_github.py` example is also demonstrating this.)

In the Ruby client we just [append `start-date`/`end-date` to the search query string](https://github.com/twingly/twingly-search-api-ruby/blob/5ac4516a803776efec18f78e3c60feedf83d28dc/lib/twingly/search/query.rb#L81-L82) before making a request, instead of sending them as URL parameters as we did before.